lib/IRGen/GenMeta.cpp

```diff
@@ -4967,6 +4967,35 @@ void irgen::emitClassMetadata(IRGenModule &IGM, ClassDecl *classDecl,
   }
 }
 
+void irgen::emitEmbeddedClassMetadata(IRGenModule &IGM, ClassDecl *classDecl,
+                                      const ClassLayout &fragileLayout) {
+  assert(!classDecl->isForeign());
+  PrettyStackTraceDecl stackTraceRAII("emitting metadata for", classDecl);
```

Comment:
Can you put this above the assertion? If the assertion fires, might be nice to have this diagnostic :)

Reply:
done!